homestead/aliases

```diff
@@ -0,0 +1,264 @@
+alias ..="cd .."
```

Comment:
Could we delete this file as it is Laravel specific?

Reply:
I thnk we could remove 99% of the `aliases` file contents. But I hesitant in doing so as this seems to be a Laravel Homestead native file and might cause issues in future upgrades of Homestead.

https://github.com/laravel/homestead/blob/master/resources/aliases